app.js

```diff
@@ -21,6 +22,7 @@ class Application{
         url: '/openapi.yaml'
       }
     }
+    expressApp.use('/api/health', healthCheck());
     expressApp.use('/api-docs', swaggerUi.serve, swaggerUi.setup(null, options));
```

Comment:
Can we align the endpoints here? /api/health & /api/docs?

Reply:
of course!